app/views/dashboards/analytics.erb

```diff
@@ -2,6 +2,31 @@
 
 <main id="main-content">
   <div class="dashboard-container analytics-container crayons-layout" id="user-dashboard">
+    <% if @organizations.any? %>
+      <nav aria-label="Organizations menu">
+        <ul class="organizations-menu">
+          <li>
+            <a href="/dashboard" class="rounded-btn crayons-link--block rounded-btn--transparent">👈 Back to Main Dashboard</a>
+          </li>
+          <li>
+            <a class="rounded-btn crayons-link--block organization <%= "active" unless params[:org_id] %>" href="<%= dashboard_analytics_path %>">
```

Comment:
When the `active` class is added, that means we're on that page currently, right? If so, I think instead of using the `active` class, we should add `aria-current="page"` if we match this condition.

Then in the code we could detect if we're on the organization page like:

`document.querySelector('.organization')?.getAttribute('aria-current') === "page";`
(in the `analyticsDashboard.js` file)

It's good practice for accessibility to add `aria-current="page"` to a link when it points to the page we're currently on 🙂

Reply:
Thanks, @aitchiss, I wasn't aware of that one!